docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md

```diff
@@ -161,15 +157,15 @@ Follow the steps below to provide the location of DB nodes. All these fields are
 
 ![Configure On-Premises Cloud Provider](/images/ee/onprem/configure-onprem-3.png)
 
-## Step 2. Provision the YugabyteDB nodes
+## Provision the YugabyteDB nodes
 
 After finishing the cloud provider configuration, click Manage Instances to provision as many nodes as your application requires.
 
 ### Add nodes
 
 ![Configure On-Premises Cloud Provider](/images/ee/onprem/configure-onprem-4.png)
 
-**For each node you want to add**, click Add Instances to add a YugabyteDB node. You can use DNS names or IP addresses when adding instances. (Instance ID is an optional user-defined identifier.)
+For each node you want to add, click Add Instances to add a YugabyteDB node. You can use DNS names or IP addresses when adding instances. (Instance ID is an optional user-defined identifier.)
```

Comment:
None of the UI element / field name refs are in bold, so I either have to format all of them or nothing. I choose the latter because I don't have time.